crates/fluvio-version-manager/src/command/itself/uninstall.rs

```diff
@@ -0,0 +1,56 @@
+use std::fs::remove_dir_all;
+
+use clap::Parser;
+use color_eyre::eyre::Result;
+use color_eyre::owo_colors::OwoColorize;
+use dialoguer::Confirm;
+use dialoguer::theme::ColorfulTheme;
+
+use crate::GlobalOptions;
+
+use crate::common::notify::Notify;
+use crate::common::workdir::fvm_workdir_path;
+
+#[derive(Clone, Debug, Parser)]
+pub struct UninstallOpt {
+    #[command(flatten)]
+    global_opts: GlobalOptions,
+}
+
+impl UninstallOpt {
+    pub async fn process(&self) -> Result<()> {
+        // Checks if FVM is already installed
+        let workdir_path = fvm_workdir_path()?;
+
+        if workdir_path.exists() {
+            if Confirm::with_theme(&ColorfulTheme::default())
+                .with_prompt(&format!(
+                    "Are you sure you want to uninstall FVM from {}?",
+                    workdir_path.display().italic()
+                ))
+                .interact()?
+            {
+                remove_dir_all(&workdir_path)?;
```

Comment:
probably have error handling here to give warning message if remove dir didn't work

Reply:
Sure! `color_eyre` will make it easier to debug as well by providing spantrace and error messages by layer.

Reply:
Achually the `?` will short circuit and return early in this case, with the error message, adding another message on screen would be redundant no?

Reply:
just catch and provide user friendly message